flake.nix

```diff
@@ -2,15 +2,16 @@
   description = "(pcb-rnd is a modular PCB layout editor)";
 
   # Nixpkgs / NixOS version to use.
-  inputs.nixpkgs.url = "nixpkgs/nixos-20.03";
+  inputs.nixpkgs.url = "nixpkgs/nixos-unstable";
 
   # Upstream source tree(s).
-  inputs.pcb-rnd-src = { type = "tarball"; url = "http://repo.hu/projects/pcb-rnd/releases/pcb-rnd-2.2.3.tar.bz2"; flake = false; };
+  inputs.pcb-rnd-src = { type = "tarball"; url = "http://repo.hu/projects/pcb-rnd/releases/pcb-rnd-3.0.0.tar.bz2"; flake = false; };
+  inputs.librnd-src = { type = "tarball"; url = "http://repo.hu/projects/librnd/releases/librnd-3.0.0.tar.bz2"; flake = false; };
 
-  outputs = { self, nixpkgs, pcb-rnd-src }:
+  outputs = { self, nixpkgs, pcb-rnd-src, librnd-src }:
     let
 
-      version = "2.2.3";
+      version = "3.0.0";
 
       # System types to support.
       supportedSystems = [ "x86_64-linux" ];
```

Comment:
they do seem to support more systems than just linux
http://cgi.repo.hu/cgi-bin/minisvn.cgi?cmd=browse&repo=pcb-rnd&path=trunk/doc
I'm happy to test darwin.

Reply:
Added darwin